lib/clickhouse_ecto.ex

```diff
@@ -15,12 +15,12 @@ defmodule ClickhouseEcto do
   def autogenerate(:binary_id), do: Ecto.UUID.generate()
   def autogenerate(type), do: super(type)
 
-  def dumpers({:embed, _} = type, _), do: [&Ecto.Adapters.SQL.dump_embed(type, &1)]
+  def dumpers({:embed, _} = type, _), do: [&Ecto.Type.embedded_dump(type, &1, :json)]
   def dumpers(:binary_id, _type), do: []
   def dumpers(:uuid, _type), do: []
   def dumpers(ecto_type, type), do: [type, &encode(&1, ecto_type)]
 
-  def loaders({:embed, _} = type, _), do: [&Ecto.Adapters.SQL.load_embed(type, &1)]
```

Comment:
`Ecto.Adapters.SQL.load_embed` was dropped from **ecto_sql**
They switched to using `Ecto.Type.embedded_load/3` up until version 3.5.0 were they completely dropped loaders/dumpers for `:embed` types. 

There were two options: 
  * continue using `Ecto.Type.embedded_dump/3` , `Ecto.Type.embedded_load/3`
  * drop those two lines of code out of the codebase.